components/ui-api-layer/internal/gqlschema/schema.graphql

```diff
@@ -374,6 +374,38 @@ type Deployment {
     boundServiceInstanceNames: [String!]
 }
 
+type Pod {
+    name: String!
+    nodeName: String!
+    namespace: String!
+    restartCount: Int!
+    creationTimestamp: Timestamp!
+    labels: Labels!
+    status: PodStatusType!
+    containerStates: [ContainerState!]!
```

Comment:
States or statuses? I see Kubernetes API has `ContainerStatuses` and we are in most cases consistent with API.

Reply:
`ContainerState` is a field in `ContainerStatus`, it seemed that we didn't need to return whole `ContainerStatus`, that's why I'm returning only properly prepared States.